src/magic_spell_effect.cpp

```diff
@@ -481,6 +481,14 @@ static void damage_targets( const spell &sp, Creature &caster,
                             const std::set<tripoint> &targets )
 {
     bool sound_played = false;
+    unsigned int affected = 0;
+    for( const auto &target : targets ) {
+        // figure out the number of targets we're dealing damage to
+        Creature *const cr = g->critter_at<Creature>( target );
+        if( cr ) {
+            affected++;
+        }
+    }
```

Comment:
```suggestion
    const int affected = std::ranges::count_if( targets, [&]( const auto &target ) { return g->critter_at<Creature>( target ) != nullptr; });
```
could use https://en.cppreference.com/w/cpp/algorithm/ranges/count.html